config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'items/new'
+  resources :items
```

Comment:
I'm curious to how the routes file with the Devise login will work with this one as we have a main page and a separate database as well.